tests/framework/installer/environment.go

```diff
@@ -43,6 +44,7 @@ func init() {
 	flag.StringVar(&Env.Helm, "helm", "helm", "Path to helm binary")
 	flag.StringVar(&Env.RookImageName, "rook_image", "rook/ceph", "Docker image name for the rook container to install, must be in docker hub or local environment")
 	flag.StringVar(&Env.ToolboxImageName, "toolbox_image", "rook/ceph", "Docker image name of the toolbox container to install, must be in docker hub or local environment")
+	flag.StringVar(&Env.BaseTestDir, "base_test_dir", "/data", "Base test directory, for use only when kubernetes master is running on localhost")
```

Comment:
The default is now optimized for the dev who is running on a different host than the tests, but this causes the CI to fail. What about the following that will still give preference to the dev scenario and get the CI working?
- Set this base_test_dir to blank from the CI. This can be done from an env var from the jenkinsfile similar to [this](https://github.com/rook/rook/blob/master/Jenkinsfile#L184).
- In installer.go, if the path is blank, call `os.Getwd()` as before

Reply:
Yeah, I was thinking along the same lines once I saw the CI failure. Thanks for the pointer to the appropriate insertion point in the CI.